simulation/costs.py

```diff
@@ -296,3 +297,35 @@ def estimate_rhi_annual_payment(
     )
 
     return rhi_annual_payment_gbp
+
+
+def estimate_boiler_upgrade_scheme_grant(
+    heating_system: HeatingSystem,
+    model: "DomesticHeatingABM",
+):
+
+    if heating_system not in [
+        HeatingSystem.HEAT_PUMP_AIR_SOURCE,
+        HeatingSystem.HEAT_PUMP_GROUND_SOURCE,
+    ]:
+        return 0
+
+    boiler_upgrade_funding_cap_gbp = 450_000_000
```

Comment:
I think we can save this one to a future PR -- I suspect the cap will not be hit, but if it does get hit, it could be interesting to explore the `boiler_upgrade_funding_cap_gbp` as an input arg / variable parameter.